### PR TITLE
[WIP] FAQ New design

### PR DESCRIPTION
**Pull request by ashishmangla0**

### CODE REVIEW

File: css/common/main.css [hunk 1]
```diff
@@ -84,4 +90,65 @@ footer {
   background-color: #030c4c;
   transition: background-color 0.2s ease-in;
   outline: #041484;
+}
+
+.container{
+  max-width: 1170px;
```
Comment (ankushdharkar) [inline css/common/main.css:96]: Where do we get this number from?

File: css/common/main.css [hunk 2]
```diff
@@ -84,4 +90,65 @@ footer {
...
+  src:  url('../../fonts/rds.eot?kckyxh');
+  src:  url('../../fonts/rds.eot?kckyxh');
+  src:  url('../../fonts/rds.eot?kckyxh#iefix') format('embedded-opentype'),
+    url('../../fonts/rds.ttf?kckyxh') format('truetype'),
+    url('../../fonts/rds.woff?kckyxh') format('woff'),
+    url('../../fonts/rds.svg?kckyxh#rds') format('svg');
+  font-weight: normal;
+  font-style: normal;
+  font-display: block;
+}
```
Comment (ankushdharkar) [inline css/common/main.css:125]: Why did we not stick with Google fonts?

File: css/common/main.css [hunk 3]
```diff
@@ -84,4 +90,65 @@ footer {
...
+
+.icon {
+  /* use !important to prevent issues with browser extensions that change fonts */
+  font-family: 'rds' !important;
```
Comment (ankushdharkar) [inline css/common/main.css:129]: I think we can move this to a separate PR

File: css/pages/faq.css [hunk 4]
```diff
@@ -3,14 +3,10 @@ html {
...
-body {
-  font-family: 'Roboto', sans-serif;
-}
-
 section {
-    border:2px solid blue;
+    /* border:2px solid blue;
     padding: 2%;
-    width :80%;
+    width :80%; */
```
Comment (ankushdharkar) [inline css/pages/faq.css:9]: Let's remove dead code please

File: css/common/main.css (hunk 1 above)
Comment (ashishmangla0) [reply css/common/main.css:96]: it's bootstrap 3.3.7 container width.

File: css/common/main.css (hunk 2 above)
Comment (ashishmangla0) [reply css/common/main.css:125]: it's not google fonts this is for icons.

**Comment by ankushdharkar** [comment]
Closed in favour of #43
